types/auto/modules/RTC/TraceablePeerConnection.d.ts

```diff
@@ -23,7 +23,7 @@
  *
  * @constructor
  */
-export default function TraceablePeerConnection(rtc: RTC, id: number, signalingLayer: any, pcConfig: object, constraints: object, isP2P: boolean, options: {
+export default function TraceablePeerConnection(rtc: any, id: number, signalingLayer: any, pcConfig: object, constraints: object, isP2P: boolean, options: {
```

Comment:
Removing RTC import from `TraceablePeerConnection.js` results in missing type here.. not sure if we can do something about that right now, thoughts?